### PR TITLE
Completed lesson 3 - Express names

### PR DESCRIPTION
**Pull request by evo777**
Able to print names that are not mine.

### CODE REVIEW

**Review by seemcat — APPROVED** [review]

File: express-GET/server.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+app.get('/llip', (req, res) => {
+  const getName = (err, file) => {
+    const result = [];
+    file.forEach(name => {
+      if (name !== 'emmanuel') {
+        result.push(name);
+      }
+    });
+    res.send(result+'\r\n');
```
Comment (songz) [inline express-GET/server.js]: whats the point of sending back `song emmanuel joe \r\n`? Like why put `\r\n` at the very end of the string?

File: express-GET/server.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+const express = require('express');
+const fs = require('fs');
+
+const port = 3000;
+const app = express();
+
+
```
Comment (allopez7) [inline express-GET/server.js]: Delete this extra line.

**Review by allopez7 — CHANGES_REQUESTED** [review]
Check comment.

File: express-GET/server.js (hunk 1 above)
Comment (evo777) [reply express-GET/server.js]: I meant to put `.join('\r\n')`. I fixed it.

File: express-GET/server.js (hunk 2 above)
Comment (evo777) [reply express-GET/server.js]: Fixed.

**Review by allopez7 — APPROVED** [review]
Good job!

**Pull request merged**